<launch>
<!-- USB_cam -->
   <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
     <param name="video_device" value="/dev/video0" />
    <param name="camera_info_url" value="package://gimbal/config/gim3_x1.yaml"/>
      <param name="image_width" value="1280" />
     <param name="image_height" value="720" />
     <param name="pixel_format" value="mjpeg" />
     <param name="camera_frame_id" value="usb_cam" />
     <param name="io_method" value="mmap"/>
     <param name="contrast" value="100"/>
     <param name="brightness" value="50"/>
     <param name="autofocus" value="false"/>


   </node>

   <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
     <remap from="image" to="/usb_cam/image_raw"/>
     <param name="autosize" value="true" />
   </node>
</launch>
